pkg/apis/core/types.go

```diff
@@ -579,6 +605,20 @@ type PersistentVolumeClaimStatus struct {
 	// +featureGate=RecoverVolumeExpansionFailure
 	// +optional
 	ResizeStatus *PersistentVolumeClaimResizeStatus
+	// volumeAttributesClassName is the name of VolumeAttributesClass which is latest applied by
+	// the external-provisioner or external-resizer. This field is not set by default but when
+	// modification is complete, it's set same as VolumeAttributesClassName in spec.
+	// This is an alpha field and requires enabling VolumeAttributesClass feature.
+	// +featureGate=VolumeAttributesClassName
+	// +optional
+	VolumeAttributesClassName string
```

Comment:
This field should also be a pointer. VolumeAttributesClassName cannot be empty and this is not set by default.  So the default should be nil.